docs/source/model_design/risk_factors.rst

```diff
@@ -531,12 +531,99 @@ Difference measures relating to the whole population tend to be more useful and
 Population Attributable Fraction (PAF)
 ++++++++++++++++++++++++++++++++++++++
 
-The population attributable fraction is a quantification of the proportion of 
-of a given cause outcome, such as cases, deaths, or DALYs, that could be 
-eliminated by removing a risk exposure.
+Measures of population impact estimate the expected impact on a population of removing or changing the distribution of risk factors in that population. In GBD, this is the level of exposure that is equal to the TMREL. They take into account the both the strength of the association (estimated by a measure of effect, like the rate ratio) **and** the distribution of the risk factor in the population. It assumes that we have established that the association between disease and risk factor is *causal*. If this assumption is true, population impact estimates measure how much of the disease in the population is caused by the suspected risk factor. 
```

Comment:
second sentence is unclear, what is the level of exposure?  fourth paragraph is unclear, what is "it" that assumes this?

Reply:
edited for clarity